linux_sql/sql/ddl.sql

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Make sure your script changes to the correct database before creating the tables